airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/data/TimeStringToIntegral.kt

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.data
+
+import io.airbyte.cdk.load.message.DestinationRecord
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalTime
+import java.time.OffsetTime
+import java.time.ZoneOffset
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+import java.time.temporal.ChronoUnit
+
+class TimeStringTypeToIntegralType : AirbyteSchemaIdentityMapper {
```

Comment:
I think this file is a duplicate of TimeStringToInteger? (same for TimeStringToIntegralTest / TimeStringToIntegerTest)

Reply:
Rebase artifact, I'm going through and removing all of these

Reply:
I think `rerere` got confused at some point, because changes I keep removing keep coming back?

Reply:
maybe - I don't think I've seen that myself, but definitely plausible. git should log something when rerere activates, if you want to dig into it (iirc something like `replaying recorded resolution`)

otherwise I'm pretty sure you can just `rm -rf .git/rr-cache` to nuke it